themes/default/content/blog/introducing-the-puluminaries/index.md

```diff
@@ -0,0 +1,103 @@
+
+### Joshua Studt
+
+<img src="joshua-studt.png" width="200" height="200" alt="Joshua Studt"/>
+<a data-track="github" href="https://github.com/orionstudt"><i class="fab fa-github"></i></a>
+
+Joshua is a longtime member of the Pulumi community, and he contributed to the implementation of Automation API for C#.
+
+### Kat Morgan
+
+<img src="kat-morgan.png" width="200" height="200" alt="Kat Morgan"/>
+<a data-track="twitter" href="https://twitter.com/usrbinkat"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/usrbinkat"><i class="fab fa-github"></i></a>&nbsp;<a data-track="linkedin" href="https://linkedin.com/in/usrbinkat"><i class="fab fa-linkedin"></i></a>
+
+Kat has been a helpful member of the Pulumi community - not only providing feedback on real-world use of Pulumi, but also sharing her knowledge with [workshops and webinars](https://konghq.com/webinars/devmyops-deploy-kong-with-pulumi?utm_souce=pulumi&utm_medium=pulumi)
+
+### Komal Ali
+
+<img src="komal-ali.png" width="200" height="200" alt="Komal Ali"/>
+<a data-track="twitter" href="https://twitter.com/zwitkali"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/komalali"><i class="fab fa-github"></i></a>&nbsp;<a data-track="linkedin" href="https://linkedin.com/in/komal-ali"><i class="fab fa-linkedin"></i></a>
+
+In addition to being a Puluminary, Komal is also a Pulumni, having spent time working to make Pulumi a better tool for developers - especially bringing to life Pulumi's Automation API!
+
+### Paul Hicks
+
+<img src="paul-hicks.png" width="200" height="200" alt="Paul Hicks"/>
+<a data-track="github" href="https://github.com/tenwit"><i class="fab fa-github"></i></a>
+
+Paul is one of the most prolific helpers in [Pulumi Community Slack](https://slack.pulumi.com/)! Chances are, if you see a great answer to a question in Slack, it came from Paul.
+
+### Ringo De Smet
+
+<img src="ringo-smet.png" width="200" height="200" alt="Ringo De Smet"/>
+<a data-track="twitter" href="https://twitter.com/ringods"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/ringods"><i class="fab fa-github"></i></a>
+
+Ringo is one of the founding board members of [the Pulumiverse](https://www.pulumi.com/blog/2022-03-30-introducing-pulumiverse/) - a place to interact and collaborate on Pulumi-based libraries, projects, and educational materials. Ringo (along with fellow Puluminary Simen A. W. Olsen) have brought their vision for expanding the Pulumi community to life with the creating of this community-operated place for anyone to communicate, contribute, and collaborate with other members of the Pulumi community and build abstractions and SDKs that can be shared and used by anyone. Ringo also created the [Pulumi Resource Type for Concourse](https://github.com/ringods/pulumi-resource).
```

Comment:
I'd personally mention here how Ringo is ever present in every community he's part of as he championed Terrafarm too